### DISCUSSION TITLE
Progress Bar

### DISCUSSION THREAD

**Discussion by lokeshwarobuli** [mantinedev/mantine#2757] [Q&A] [answered]
`<div style={{display:"flex"}}><Progress value={50}/></div>`

In the scenario above, the progress bar is not being rendered inside the div, can you please help me out
When I remove the display flex, the progress bar appears, but if I add it back it doesn't appear on the UI

Thanks in Advance

**Comment by rtivital** [comment, marked as answer]
set width on progress: `<Progress sx={{ width: 200 }} />`, of flex: `<Progress sx={{ flex: 1 }} />`

**Reply by lokeshwarobuli** [reply]
Thanks that worked
